src/axom/core/execution/internal/cuda_exec.hpp

```diff
@@ -13,11 +13,11 @@
 #include "umpire/Umpire.hpp"
 
 #ifndef RAJA_ENABLE_CUDA
-#error *** CUDA_EXEC requires a CUDA enabled RAJA ***
+#error "*** CUDA_EXEC requires a CUDA enabled RAJA ***"
```

Comment:
Here and in similar places, the quotes are not needed for the argument to the `#error` directive. I think this would actually print the error message with the `"`, which is not the intent.  Please, revert this change.

Reply:
Thanks @gzagaris.  I added them because clang-format removed the spaces between the `#error` and `***` and it became
```
 #error*** CUDA_EXEC requires a CUDA enabled RAJA *** 
```

What do you suggest?
* Adding quotes ?
* Removing the askerisks ?
* \<other> ?

Reply:
I would remove the asterisks and leave out the quotes. If it is hit, it will still print out the message indicating an error.

Reply:
Done.  I removed the quotes and the asterisks,